charts/logprep/values.yaml

```diff
@@ -0,0 +1,111 @@
+#   credentials:
+#     name: logprep-credentials # Name of the secret containing the logprep credentials file
+#   imagePullSecret:
+#     name: logprep-image-pull-secret # Name of the secret containing the image pull secret
+secrets: {}
+
+# Boolean to signal to use affinity to avoid deploying multiple instances of the
+# pod on the same node
+affinity: false
+
+# inject extra labels to all resources metadata
+extraLabels: {}
+
+# Use this to annotate the logprep pods
+# podAnnotations:
+#   sidecar.istio.io/rewriteAppHTTPProbers: "false"
+podAnnotations: {}
+
+# If enabled, the exporter will be started on the specified port and
+# the metrics service and a prometheus PodMonitor will be deployed.
+# You have to ensure, that the prometheus operator is installed and
+# the PodMonitor resource is available.
+exporter:
+  enabled: true
+  port: 8000
+  service_port: 8001
+  scrape_interval: 30s
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+
+# Logprep logging configuration.
+# See: https://logprep.readthedocs.io/en/latest/user_manual/configuration/index.html#configuration-file-structure
+# for available configuration options.
+logger:
+  level: DEBUG
+
+# The logprep input connector configuration
+# Note: The Name of the input is derived from the `type` of the input.
+# In example, if the type is `file_input`, the name of the input will be `file`.
+#
+# If the type is `http_input`, an extra service will be populated and the readiness
+# probe will be set to the health check of the configured http input.
+#
+# See https://logprep.readthedocs.io/en/latest/user_manual/configuration/input.html
+# for available configuration options.
+input: {}
+
+# The logprep output connector configuration
+# Note: This is only the default output. Additional outputs can be configured in
+# the configurations section. The Name of the default output is derived from the
+# `type` of the output. In example, if the type is `elasticsearch_output`, the
+# name of the output will be `elasticsearch`. Keep this in mind if you specify
+# additional outputs in the configurations section.
+# See https://logprep.readthedocs.io/en/latest/user_manual/configuration/output.html
+# for available configuration options.
+output: {}
+
+# Additional logprep configurations
+# Note: The configuration keys 'logger', 'metrics', 'input' and 'output' will overwrite the
+# corresponding keys in these configurations as they will be merged into
+# the final configuration.
+# for configurations with data from an external API put the url in the name field and leave the data field undefined
+#
+# Example:
+#
+# configurations:
+#   - name: logprep-config
+#     data:
+#       pipeline: []
+#   - name: http://bla-bla.bla
+#
+# See https://logprep.readthedocs.io/en/latest/user_manual/configuration/index.html#configuration-file-structure
+# for available configuration options.
+configurations:
+  - name: logprep-config
+    data:
+      pipeline: []
```

Comment:
I was able to reproduce this but this can't be resolved by helm. I added a converter to the attrs field to ensure this is parsed to float.

you can test it with:

```bash
helm install logprep ./charts/logprep --set "image.tag=py3.11-dev-add-helm-chart"
```